Detectors/GRP/workflows/src/GRPDCSDPsSpec.cxx

```diff
@@ -0,0 +1,247 @@
+    for (auto& i : *dpid2DataDesc) {
+      vect.push_back(i.first);
+    }
+  } else {
+    LOG(info) << "Configuring via hardcoded strings";
+    std::vector<std::string> aliasesBFieldDouble = {"L3Current", "DipoleCurrent"};
+    std::vector<std::string> aliasesBFieldBool = {"L3Polarity", "DipolePolarity"};
+    std::vector<std::string> aliasesEnvVar = {"CavernTemperature", "CavernAtmosPressure", "SurfaceAtmosPressure", "CavernAtmosPressure2"};
+    std::vector<std::string> compactAliasesLHCIFDouble = {"LHC_IntensityBeam[1..2]_totalIntensity", "ALI_Background[1..3]",
+                                                          "ALI_Lumi_Total_Inst",
+                                                          "BPTX_deltaT_B1_B2", "BPTX_deltaTRMS_B1_B2",
+                                                          "BPTX_Phase_B[1..2]", "BPTX_PhaseRMS_B[1..2]", "BPTX_Phase_Shift_B[1..2]"};
+    std::vector<std::string> aliasesLHCIFDouble = o2::dcs::expandAliases(compactAliasesLHCIFDouble);
+    std::vector<std::string> aliasesLHCIFString = {"ALI_Lumi_Source_Name", "MACHINE_MODE", "BEAM_MODE"};
+    std::vector<std::string> aliasesLHCIFCollimators = {"LHC_CollimatorPos_TCLIA_4R2_lvdt_gap_downstream", "LHC_CollimatorPos_TCLIA_4R2_lvdt_gap_upstream",
+                                                        "LHC_CollimatorPos_TCLIA_4R2_lvdt_left_downstream", "LHC_CollimatorPos_TCLIA_4R2_lvdt_left_upstream",
+                                                        "LHC_CollimatorPos_TCLIA_4R2_lvdt_right_downstream", "LHC_CollimatorPos_TCLIA_4R2_lvdt_right_upstream"};
+
+    for (const auto& i : aliasesBFieldDouble) {
+      vect.emplace_back(i, o2::dcs::RAW_DOUBLE);
+    }
+    for (const auto& i : aliasesBFieldBool) {
+      vect.emplace_back(i, o2::dcs::RAW_BOOL);
+    }
+    for (const auto& i : aliasesEnvVar) {
+      vect.emplace_back(i, o2::dcs::RAW_DOUBLE);
+    }
+    for (const auto& i : aliasesLHCIFDouble) {
+      vect.emplace_back(i, o2::dcs::RAW_DOUBLE);
+    }
+    for (const auto& i : aliasesLHCIFCollimators) {
+      vect.emplace_back(i, o2::dcs::RAW_DOUBLE);
+    }
+    for (const auto& i : aliasesLHCIFString) {
+      vect.emplace_back(i, o2::dcs::RAW_STRING);
+    }
+  }
+
+  LOG(info) << "Listing Data Points for GRP:";
+  for (auto& i : vect) {
+    LOG(info) << i;
+  }
+
+  mProcessor = std::make_unique<o2::grp::GRPDCSDPsProcessor>();
+  bool useVerboseMode = ic.options().get<bool>("use-verbose-mode");
+  LOG(info) << " ************************* Verbose?" << useVerboseMode;
+  if (useVerboseMode) {
+    mProcessor->useVerboseMode();
+  }
+  mProcessor->init(vect);
+  mTimer = HighResClock::now();
+}
+//__________________________________________________________________
+
+void GRPDCSDPsDataProcessor::run(o2::framework::ProcessingContext& pc)
+{
+  auto tfid = o2::header::get<o2::framework::DataProcessingHeader*>(pc.inputs().get("input").header)->startTime;
```

Comment:
`startTime` is a dummy counter, it cannot be used as an alias for the time. Moreover, when it comes from the dcs-proxy, where the TF has no usual meaning, it even cannot be converted to the abs. time.
If you want, I can change the dcs2dpl to send 'now()' in the `DataProcessingHeader::creation`, then you can use 
```
auto creationTime = DataRefUtils::getHeader<DataProcessingHeader*>(pc.inputs().getFirstValid(true))->creation;
```
directly for the time. Alternatively, you can simply use `now()` here. But the most precise would be extracting the timestamp from the DP itself.

Reply:
Hi Ruben, 
This was the plan, I started with `startTime` more as a placeholder (see also our other mail exchange). I planned to add a comment to the PR, but I forgot :-) 
I would use for each object probably the oldest time of all datapoints, or, as you say, "now". 
This is something that should be changed in several places, which is also why I kind of postponed it.

Reply:
I decided to not use the timestamps of the measurements because they might be out of sync, and they could give the wrong message. 
I am using now "creation", and if this is -1, then I use "now". 
But I think we should set it in the proxy, same for the configurations.

Reply:
Then simply use 
```
auto creationTime = DataRefUtils::getHeader<DataProcessingHeader*>(pc.inputs().getFirstValid(true))->creation;
```
I will add its filling to the proxy today.

Reply:
OK, I now have:

```
auto startValidity = o2::header::get<o2::framework::DataProcessingHeader*>(pc.inputs().get("input").header)->creation;
```

but I can change it to the one above (I guess they are equivalent, no?).

Reply:
done.